api/src/main/java/org/pmiops/workbench/monitoring/views/OpenCensusView.java

```diff
@@ -63,9 +63,10 @@ default MeasureDouble getMeasureDouble() {
 
   Aggregation getAggregation();
 
+  // TODO(jaycarlton) What is this for?
   List<TagKey> getColumns();
 
-  default View toOpenCensusView() {
+  default View toView() {
```

Comment:
nitpick: given the conversions going on, I think the prior method name is worth the extra chars.

Reply:
No one will ever need to call this directly, so it's just a question of clarity. The real problem here is file name. I may rename it `Metric`.